server/src/main/java/io/spine/server/event/model/EventAcceptingMethodParams.java

```diff
@@ -95,14 +92,14 @@
     private final ImmutableList<Class<?>> expectedParameters;
     private final boolean awareOfCommandType;
 
-    EventAcceptingMethodParams(ImmutableList<Class<?>> parameters, boolean type) {
-        expectedParameters = parameters;
-        awareOfCommandType = type;
+    EventAcceptingMethodParams(boolean awareOfCommandType, Class<?>... parameters) {
```

Comment:
While we're here, please consider replacing this `boolean` flag with an enum for better readability.